composer.json

```diff
@@ -29,8 +29,8 @@
     "symfony/process": "~2.1||~4.1.0",
     "symfony/yaml": "^3.3||^4.0",
     "twig/twig": "^1.0||^2.0",
-    "magento/magento-cloud-components": "^1.0.1",
-    "magento/magento-cloud-patches": "^1.0.0"
+    "magento/magento-cloud-components": "~1.0.1",
```

Comment:
Please change to "<1.0.3", since we don't want to update components for 2002.0.x beyond 1.0.2 (+ possible patches).
Or you feel confident enough that we won't introduce huge breaking changes later?

Reply:
Breaking changes should go under the new minor version.